salt/client/ssh/__init__.py

```diff
@@ -896,9 +896,10 @@ def run_wfunc(self):
             opts_pkg['id'] = self.id
 
             retcode = 0
-
+            popts = copy.copy(self.context['master_opts'])
```

Comment:
ok, I changed it to a deepcopy, makes sense to be safe, my thinking is that since it is only the top level keys being changes then copy would be enough, but it is better to be safe